internal/asm/f32/ge_test.go

```diff
@@ -0,0 +1,345 @@
+// Copyright ©2017 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build go1.7
+
+package f32
+
+import (
+	"fmt"
+	"math"
+	"testing"
+
+	"gonum.org/v1/gonum/blas/testblas"
+)
+
+var gerTests = []struct {
+	name string
+	a    []float32
+	m    uintptr
+	n    uintptr
+	x    []float32
+	y    []float32
+
+	want []float32
+}{
+	{
+		name: "Unit",
```

Comment:
I like how in `f64` each test case has a comment with what kernels are exercised by the test. Could you do it here also? I'd like to make sure that all kernels are covered by the tests.